bin/cli_instrument.sh

```diff
@@ -37,12 +37,13 @@ log_output=$(slim instrument \
   $extraArgs \
   $BASEIMAGE 2>&1 | tee /dev/stderr)
 
-workflow_id=$(echo "$log_output" | grep -Eo 'workflow id: [a-zA-Z0-9\.]+')
-workflow_id=${workflow_id#"workflow id: "}
-
-if [[ $log_output =~ "[instrument] instrumented" ]]; then
+if [[ "$?" == "0" ]]; then
```

Comment:
Oh, it's much easier now - just `workflow_id=$(slim instrument ...)` should do. The human-readable output goes to stderr and only the workflow ID is written to stdout.